app/main/views/briefs.py

```diff
@@ -198,12 +198,8 @@ def redirect_to_next_page():
     # Unformat brief response into data for form
     service_data = question.unformat_data(brief_response)
 
-    # This line can soon be removed when we no longer show boolean list questions as part of the new flow
-    section.inject_brief_questions_into_boolean_list_question(brief)
-
```

Comment:
This looks good.

Any chance we can get rid of `inject_brief_questions_into_boolean_list_question` entirely, or do we need to keep the old flow alive for some reason?